### PR TITLE
Implementing an Image Classification App in One Minute for Tinyms v0.3.1

### PR DESCRIPTION
**Pull request by xinwenh**
**What type of PR is this?**

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

> `/kind enhancement`</br>

**What does this PR do / why do we need it**:
This PR updates tinyms0 3.1 realize the document of "realizing one graphic classification application in one minute" and publish it (Chinese version and English version) on the official document website.

**Which issue(s) this PR fixes**:

**Special notes for your reviewers**:

### CODE REVIEW

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 1]
```diff
@@ -7,18 +7,13 @@
...
-    " - Flask: `1.1.2`\n",
-    " - MindSpore: `CPU-1.1.1`\n",
-    " - TinyMS: `0.1.0`\n",
-    " - numpy: `1.17.5`\n",
-    " - Pillow: `8.1.0`\n",
-    " - pip: `21.0.1`\n",
-    " - requests: `2.18.4`\n",
+    " - docker: `v18.06.1-ce`\n",
+    " \n",
+    " 使用jupyter需要到[TinyMS官方仓项目](https://tinyms.readthedocs.io/zh_CN/latest/quickstart/install.html)获取docker拉取tinyms0.3.0镜像的指令\n",
```
Comment (hellowaywewe) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:16]: Hi, @xinwenh Please keep the format uniform, for example: Jupyter: 使用tinyms 0.3.1-jupyter容器，可参考。。。部署该环境。

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 2]
```diff
@@ -245,7 +240,53 @@
...
+    "\n",
+    " * Debug mode: off"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "691227b9",
+   "metadata": {},
+   "source": [
+    "#### 如果出现如上，说明服务器已启动，正在运行，需要新开一个jupyter继续后面的操作"
```
Comment (hellowaywewe) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:268]: The statement is not smooth, you can modify it more better.

File: docs/en/source/quickstart/quickstart_in_one_minute.ipynb [hunk 3]
```diff
@@ -11,20 +11,13 @@
...
-    " - Flask: `1.1.2`\n",
-    " - MindSpore: `CPU-1.1.1`\n",
-    " - TinyMS: `0.1.0`\n",
-    " - numpy: `1.17.5`\n",
-    " - Pillow: `8.1.0`\n",
-    " - pip: `21.0.1`\n",
-    " - requests: `2.18.4`\n",
+    " - docker: `v18.06.1-ce`\n",
+    " \n",
+    " To use jupyter,you need to go to[Docker拉取tinyms镜像](https://tinyms.readthedocs.io/zh_CN/latest/quickstart/install.html)to obtain docker and pull tinyms0 3.0 mirror instructions\n",
```
Comment (hellowaywewe) [inline docs/en/source/quickstart/quickstart_in_one_minute.ipynb:16]: Do not leave Chinese characters, pay attention to spaces. The translation should be consistent with the Chinese documents.

File: docs/en/source/quickstart/quickstart_in_one_minute.ipynb [hunk 4]
```diff
@@ -245,7 +238,53 @@
...
+    "\n",
+    " * Debug mode: off"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b394470",
+   "metadata": {},
+   "source": [
+    "#### If the following occurs, it indicates that the server has been started and is running. You need to open a new jupyter to continue the following operations"
```
Comment (hellowaywewe) [inline docs/en/source/quickstart/quickstart_in_one_minute.ipynb:266]: The translation should be consistent with the Chinese documents.

**Comment by xinwenh** [comment]
Sorry, it needs to be closed because multiple duplicate PR's have been submitted

**Pull request closed without merging**